start-site/src/main/resources/templates/maintenance-mode-dependencies.mustache

```diff
@@ -0,0 +1,6 @@
+The dependencies listed below are in maintenance mode. We do not recommend adding them to
```

Comment:
I'd start with something tailored for the spring cloud use cases with a general section that titles exactly what happens and then some blurp that summarizes the situation. Perhaps a link to blog post announcement?